crm_security_group/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module add new group in Sales to show only CRM (and exclude all about sale orders).
```

Comment:
s/add/adds, but the specifications are not this one, but a totally new module category called CRM for having 3 security groups twins to the Sales one, as we don't want to lose the possibility of restricting the visibility of the documents.

Then, you need to recreate all the ACLs, record rules, and add menu groups according these 3 new security groups.